internal/featuresupport/features.go

```diff
@@ -721,3 +721,41 @@ func (feature *MinimalIso) getFeatureActiveLevel(_ *common.Cluster, infraEnv *mo
 	}
 	return activeLevelNotActive
 }
+
+// MceFeature
+type MceFeature struct{}
+
+func (feature *MceFeature) New() SupportLevelFeature {
+	return &MceFeature{}
+}
+
+func (feature *MceFeature) getId() models.FeatureSupportLevelID {
+	return models.FeatureSupportLevelIDMCE
+}
+
+func (feature *MceFeature) getName() string {
+	return "multicluster engine"
+}
+
+func (feature *MceFeature) getSupportLevel(filters SupportLevelFilters) models.SupportLevel {
+	if !isFeatureCompatibleWithArchitecture(feature, filters.OpenshiftVersion, swag.StringValue(filters.CPUArchitecture)) {
+		return models.SupportLevelUnavailable
+	}
+
+	return models.SupportLevelSupported
+}
+
+func (feature *MceFeature) getIncompatibleFeatures() *[]models.FeatureSupportLevelID {
```

Comment:
Please suggest an edit on https://docs.google.com/spreadsheets/d/1IJugmChJTKMBjdXHojEgfsyE8WT-dGbUpETn8fhuI7g/ to update the incompatible feature and architectures

Reply:
I have requested edit access from the owner as I cannot suggest in "View only" mode